How-To/Add-columns-to-open-data-assets.md

```diff
@@ -0,0 +1,23 @@
+# Add Columns to Open Data Assets
+
+The data team maintains a motley collection of [open data assets](https://datacatalog.cookcountyil.gov/browse?tags=cook+county+assessor). The [Athena views](https://ccao-data.github.io/data-architecture/#!/overview/ccao_data_athena) that feed these assets are updated regularly with new features, and it's important to maintain parity.
+
+1. [Log in to](https://datacatalog.cookcountyil.gov/login) the open data portal using the Data team's credentials.
+2. Click "My Assets" on the left.
```

Comment:
**nitpick:** We used backticks before for differentiating actions and fields. i.e.: 

```suggestion
2. Click `My Assets` on the left.
```